app/views/events/forms/_League.html.erb

```diff
@@ -1,6 +1,20 @@
 <%= render 'events/forms/parts/dates', form: form %>
 <%= render 'events/forms/parts/player_type', form: form %>
 
+<div class="form-group" id="event_min_players_per_team">
+  <%= form.label :min_players_per_team, class: 'control-label col-lg-2' %>
+  <div class="col-lg-10">
+    <%= form.number_field :min_players_per_team, class: 'form-control', min: 2, :value => 2 %>
+  </div>
+</div>
+
+<div class="form-group" id="event_max_players_per_team">
+  <%= form.label :max_players_per_team, class: 'control-label col-lg-2' %>
+  <div class="col-lg-10">
+    <%= form.number_field :max_players_per_team, class: 'form-control', min: 2, :value => 2 %>
+  </div>
+</div>
```

Comment:
if this should only be possible for Leagues and Tournaments, I would put it in a separate partial form, since this would reduce code duplication.

Do it analogue to dates and player_type.